css-layout-classes/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <link rel="stylesheet" href="styles.css">
+  <title>css layout classes</title>
+</head>
+
+<body>
+  <div class="container">
+
+    <div class="row">
+      <div class="column-full">
+        <header><h1>Choose your character!</h1></header>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-third">
+        <img src="images/link.png" alt="Zelda, jk Link">
+        <aside aria-label="main">
+          <h2>Super Smash Bros. Ultimate</h2>
+          <p>Super Smash Bros. Ultimate received universal acclaim from both critics and players, with some critics calling it the best installment in the series. It received praise for its large amount of content and fine-tuning of existing Smash gameplay elements, although its online mode was widely criticized.</p>
+        </aside>
+        <img src="images/yoshi.png" alt="Yoshi">
+      </div>
+    </div>
+
+    <article>
```

Comment:
the article element isn't needed here, the div with the class of row will indicate the next row